azure/sdk-identity-resources-storage/AzureIdentityStorageExample.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>2eec9aab-2e9b-4211-82f7-98c5894764e9</UserSecretsId>
```

Comment:
I'm curious if this should be included in the sample, this won't work for anyone else right?